doc/tutorial.md

```diff
@@ -195,24 +195,25 @@ minlenratio: 0.3
 - The pure attention mode requires to set the maximum and minimum hypothesis length (`--maxlenratio` and `--minlenratio`), appropriately. In general, if you have more insertion errors, you can decrease the `maxlenratio` value, while if you have more deletion errors you can increase the `minlenratio` value. Note that the optimum values depend on the ratio of the input frame and output label lengths, which is changed for each language and each BPE unit.
 - About the effectiveness of hybrid CTC/attention during training and recognition, see [2] and [3]. For example, hybrid CTC/attention is not sensitive to the above maximum and minimum hypothesis heuristics. 
 
-### Transducer
+## Transducer
 
-ESPnet also supports transducer-based models.
-To switch to transducer mode, the following should be set in the training config:
+ESPnet supports models trained with transducer loss, aka transducer models. To train such model, the following should be set in the training config:
 
 ```
 criterion: loss
 model-module: "espnet.nets.pytorch_backend.e2e_asr_transducer:E2E"
 ```
 
+### Architecture
+
```

Comment:
Can you emphasize that the customizable architecture is only supported in the transducer mode and why this is important for the transducer mode?

Reply:
Is that okay for you?

Reply:
LGTM!